src/storage/src/region/writer.rs

```diff
@@ -131,6 +131,8 @@ impl RegionWriter {
         let mut action_list = RegionMetaActionList::with_action(RegionMetaAction::Edit(edit));
         action_list.set_prev_version(prev_version);
         let manifest_version = manifest.update(action_list).await?;
+        // Notify checkpointer the flushed manifest version.
+        manifest.set_flushed_manifest_version(manifest_version);
```

Comment:
correctness: What if this is a meta action created by compaction? Looks like we should not increase the flushed_manifest_version in this case.